include/swift/Option/Options.td

```diff
@@ -1785,6 +1785,10 @@ def compiler_style_diags: Flag<["-", "--"], "compiler-style-diags">,
   Flags<[NoDriverOption, SwiftAPIDigesterOption]>,
   HelpText<"Print compiler style diagnostics to stderr.">;
 
+def err_on_abi_breakage: Flag<["-", "--"], "err-on-abi-breakage">,
```

Comment:
Nit: most flags I can find that have to do with enabling errors spell out "error"

Reply:
Thanks! updated.